sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1742,20 +1747,27 @@ export interface SearchIndexClientOptions extends PipelineOptions {
 
 // @public
 export interface SearchIndexer {
+    cache?: SearchIndexerCache;
     dataSourceName: string;
     description?: string;
-    encryptionKey?: SearchResourceEncryptionKey | null;
+    encryptionKey?: SearchResourceEncryptionKey;
     etag?: string;
     fieldMappings?: FieldMapping[];
-    isDisabled?: boolean | null;
+    isDisabled?: boolean;
     name: string;
     outputFieldMappings?: FieldMapping[];
-    parameters?: IndexingParameters | null;
-    schedule?: IndexingSchedule | null;
+    parameters?: IndexingParameters;
+    schedule?: IndexingSchedule;
     skillsetName?: string;
     targetIndexName: string;
 }
 
+// @public (undocumented)
```

Comment:
missing doc comment?

Reply:
This is auto generated. So, I could not add it

Reply:
I've worked around this in the past by writing a swagger transform that adds the documentation, but also feel free to file a bug on the swagger itself. :)